components/Cards/MealCard.js

```diff
@@ -73,70 +74,72 @@ export default function MealCard({
             />
           )}
           {weekdays && <StyledDragLine />}
-          <StyledSettingsDiv>
-            {numberOfPeople !== undefined && (
-              <SetNumberOfPeople
-                numberOfPeople={numberOfPeople}
-                handleChange={(change) => changeNumberOfPeople(change)}
-                $margin="0.75rem 0 0 1.5rem"
-                reassignRecipe={reassignRecipe}
-                day={day}
-              />
-            )}
-            {reassignRecipe !== undefined && (
-              <StyledReload
-                width={30}
-                height={30}
-                onClick={() => {
-                  reassignRecipe(day);
-                }}
-              />
-            )}
-            {weekdays && (
-              <StyledMenu
-                width={30}
-                height={30}
-                onClick={() => toggleMenu(index)}
-                $rotate={menuVisible[index]}
-              />
-            )}
-            {menuVisible[index] && (
-              <MenuContainer
-                top="7.3rem"
-                right="calc(2 * var(--gap-between) + 18px)"
-                toggleMenu={() => toggleMenu(index)}
-              >
-                <UnstyledButton
-                  onClick={() => {
-                    setIsModalCollection(true);
-                    toggleMenu(index);
-                  }}
-                >
-                  <BookUser width={15} height={15} /> Rezept im Kochbuch
-                  speichern
-                </UnstyledButton>
-                <UnstyledButton
+          {!userIsHouseholdAdmin === false && (
+            <StyledSettingsDiv>
+              {numberOfPeople !== undefined && (
+                <SetNumberOfPeople
+                  numberOfPeople={numberOfPeople}
+                  handleChange={(change) => changeNumberOfPeople(change)}
```

Comment:
If you are just passing the event to the function we can simplify it a bit. 

```suggestion
                  handleChange={changeNumberOfPeople}
```